### ISSUE TITLE
Only download drivers for the current OS

### ISSUE THREAD

**Issue by vmassol** [webdriverextensions/webdriverextensions-maven-plugin#27]
It would be nice if this plugin could download only the drivers that are needed for the current OS, since that would make the build faster and it's completely unnecessary to download drivers for other OSes ;)

**Comment by andidev** [comment]
This is possible by omitting the drivers tag. 

From docs:
```
If no drivers are provided the latest drivers will be installed for the running platform and the bit version will be chosen as if it has not been provided (see rule below).

If no bit version is provided for a driver the bit will automatically be set to 32 if running the plugin on a windows or mac platform. However if running the plugin from a linux platform the bit will be determined from the OS bit version. 
```

**Comment by vmassol** [comment]
Yes but I don't need to download all drivers, only the gecko one ;)

**Comment by andidev** [comment]
Then use the following config 
```
<drivers>
  <driver>
    <name>geckodriver</name>
  </driver>
</drivers>
```

**Comment by vmassol** [comment]
ok cool didn't realized that this would work. Cool.

Now I had to specify the version and url since version 0.15.0 is not in the default list... (sent a PR that you applied for that, thxs! Would need it released to be perfect ;)).

**Comment by andidev** [comment]
I merged the repo PR so it should be available right away.

**Comment by vmassol** [comment]
ok didn't realize that the plugin required an internet connection... hmm... Thanks anyway!

**Comment by dve** [comment]
#31 fixes version selection for chromedriver on 64bit windows if you configure chromedriver like
 

```
<drivers> 
 <driver>
    <name>chromedriver</name>
  </driver>
</drivers>
```

**Comment by andidev** [comment]
Fix #31 released in version 3.1.2
